src/course-unit/breadcrumbs/Breadcrumbs.jsx

```diff
@@ -14,6 +17,17 @@ const Breadcrumbs = () => {
   const intl = useIntl();
   const { ancestorXblocks } = useSelector(getCourseSectionVertical);
   const [section, subsection] = ancestorXblocks ?? [];
+  const navigate = useNavigate();
+  const waffleFlags = useSelector(getWaffleFlags);
+
+  const handleClick = (e, url) => {
+    e.preventDefault();
+    if (waffleFlags?.useNewCourseOutlinePage) {
```

Comment:
If the waffle flags haven't loaded yet, won't this default to `false`? I think we should generally be defaulting to `true` and assuming the new things are enabled (except the unit page).

Reply:
Corrected in follow-up PR: [`885ecab` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/885ecaba6852f91719b84a62ea9b660c423b25ee)